rpcserver.go

```diff
@@ -344,6 +345,36 @@ func handleAddNode(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (in
 	return nil, nil
 }
 
+// handleNode handles node commands.
+func handleNode(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
+	c := cmd.(*btcjson.NodeCmd)
+
+	addr := normalizeAddress(c.Addr, activeNetParams.DefaultPort)
+	var err error
+	switch c.SubCmd {
+	case "disconnect":
+		err = s.server.DisconnectNode(addr)
+	case "remove":
+		err = s.server.RemoveNode(addr)
+	case "connect":
+		switch *c.ConnectSubCmd {
```

Comment:
This will panic if the optional parameter isn't specified currently.

I'd suggest something like:

```Go
subCmd := "non-perm"
if c.ConnectSubCmd != nil {
	subCmd = *c.ConnectSubCmd
}
switch subCmd {
...
}
```